packages/api/src/command/medical/document/document-query.ts

```diff
@@ -79,13 +79,11 @@ export async function queryDocumentsAcrossHIEs({
     patient,
   });
 
-  const startedAt = new Date();
-
   const updatedPatient = await storeQueryInit({
     id: patient.id,
     cxId: patient.cxId,
     cmd: {
-      documentQueryProgress: { requestId, startedAt, download: { status: "processing" } },
+      documentQueryProgress: { requestId, download: { status: "processing" } },
```

Comment:
Removing as a i need to set at after the scheduled DQ for each HIE

Reply:
Can't we just filter it out on each HIE? My understanding is that this represents when the DQ as triggered regardless of HIE, right? It makes sense to have this around, IMO.

Reply:
I mean we wont be using it for now for the purposes of analytics but i can just leave it yea